### ISSUE TITLE
Cannot specify File Input multiple attribute

### ISSUE THREAD

**Issue by msaron** [Semantic-Org/Semantic-UI-React#1386]
I tried all different ways but could not specify the following input for selecting multiple files. Single file selection works fine. How to specify the multiple property?

`<input type="file" multiple />`

**Comment by levithomason** [comment]
There is a whitelist array of HTML input props in the Input component. These props are spread in the HTML input component within the div wrapper. The multiple prop simply needs to be added to that array.

**Comment by msaron** [comment]
Maybe there should be a way of adding any additional props that makes it flexible for any future HTML5 attributes that come along?

**Comment by levithomason** [comment]
Given there are a small finite number of these props and they change very infrequently, I think it makes sense to whitelist them.

A workaround is to use the `input` prop to define the HTML inputs props:

http://react.semantic-ui.com/elements/input
```
input		{custom}	       Shorthand for creating the HTML Input.
```

```jsx
<Input input={{ multiple: true }} />

// or

<Input input={<input multiple />} />
```

You can pass a type, props object, or an element there ^ 👍

**Comment by msaron** [comment]
Thanks for the example. I did not know that you could use the Input like this.

**Comment by levithomason** [comment]
Indeed, the crux of this library is the power of our shorthand props.  They have yet to be documented #561 😄

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.67.1`.
